Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -88,6 +114,10 @@ GST_DEBUG_CATEGORY(webkit_mse_debug);
 
 namespace WebCore {
 
+unsigned MediaPlayerPrivateGStreamerMSE::mediaMaxWidth = 0;
+unsigned MediaPlayerPrivateGStreamerMSE::mediaMaxHeight = 0;
+unsigned MediaPlayerPrivateGStreamerMSE::mediaMaxFrameRate = 0;
+
```

Comment:
Add another boolean static variable to flag if parsing has been attempted (even if it was unsuccessful).